drivers/SmartThings/matter-thermostat/profiles/air-purifier-ac-rock.yml

```diff
@@ -0,0 +1,26 @@
+name: air-purifier-ac-rock
```

Comment:
Is it too late to change ac to something like carbon or cf? I just think it's going to be confusing to see "ac" and for it not to mean air conditioner

Reply:
it is too late I think. I agree, but this is what someone decided

Reply:
actually, I negate my statement. In [this superset query](https://superset.samsungiotanalytics.com/superset/sqllab?id=1155), I do not actually see any profiles using -ac yet, so we can change this.